client/web/src/components/readonly-toggle.tsx

```diff
@@ -0,0 +1,142 @@
+import cx from "classnames"
+import React, { useCallback, useState } from "react"
+import { AuthResponse, AuthType } from "src/hooks/auth"
+import { NodeData } from "src/hooks/node-data"
+import { ReactComponent as ChevronDown } from "src/icons/chevron-down.svg"
+import { ReactComponent as Eye } from "src/icons/eye.svg"
+import { ReactComponent as User } from "src/icons/user.svg"
+import Popover from "src/ui/popover"
+import ProfilePic from "src/ui/profile-pic"
+
+export default function ReadonlyToggle({
```

Comment:
This seems like a weird name for this component.  It sounds like it would let you toggle read-only mode on and off, but that's not the case.  It's really our main identity and login component.

Reply:
Renamed to `LoginToggle` for now